romana-install/roles/romana/install-master/tasks/romana_from_github.yml

```diff
@@ -0,0 +1,31 @@
+- name: Download Go
```

Comment:
Correct. This isn't for running unit tests, but for running end-to-end tests on a cluster with a specific version of the core repo.